tsconfig.json

```diff
@@ -38,9 +38,6 @@
             ],
             "@verkkokauppa/message-experience-api/*": [
                 "message-experience-api/src/*"
-            ],
-            "@verkkokauppa/auth-helsinki-profile/*": [
-                "auth-helsinki-profile/src/*"
```

Comment:
Why is this removed? Not required anymore?

Reply:
Good catch, I had some problems with these tsconfig files because windows changes the paths and I need to reset these files manually.